<launch>
  <!--<node pkg="nodelet" type="nodelet" name="camera_process" args="manager" output="screen"/>-->

  <node pkg="nodelet" type="nodelet" name="aruco_ros" args="load aruco_ros/ArucoSingle /camera_process" output="screen">
    <param name="camera_info_path" value="/camera/camera_info" />
    <param name="image_path" value="/camera/image_raw" />
    <param name="isPubDebug" value="True"/>
    <param name="image_is_rectified" value="True"/>
    <param name="switch_smallest_aruco" value="True"/>
    <param name="marker_size"        value="0.05"/>    <!-- in m -->
    <param name="marker_id"          value="0"/>
    <param name="corner_refinement"  value="LINES" /> <!-- NONE, HARRIS, LINES, SUBPIX -->

  </node>
</launch>
